includes/admin/class-sensei-learners-main.php

```diff
@@ -165,10 +165,10 @@ public function get_columns() {
 			case 'learners':
 				$columns = array(
 					'title'            => __( 'Learner', 'sensei-lms' ),
+					'enrolment_status' => __( 'Enrollment', 'sensei-lms' ),
```

Comment:
I noticed that in the design this title was renamed to "Access". Should we change it?

Reply:
Hmm not sure I agree with that one. I think that it was named 'Access' because it meant to have many different values initially but then it was decided to just display Enrolled/Not Enrolled so I think that 'Enrollment' makes more sense.